reporting/impl/src/main/resources-windup/reports/templates/include/navbar_left_brand.ftl

```diff
@@ -0,0 +1,2 @@
+<strong class="wu-navbar-header">Tackle Analysis</strong>
+<img align="right" class="wu-navbar-header" src="${redHatLogoPrefix}resources/img/tackle-horizontal-reverse.png" />
```

Comment:
Don't we have this image in SVG? It would look better in the final reports. Just a suggestion :+1:

Reply:
Unfortunately, we don't have an SVG file. Even the current icon needed to be somehow created based on another base image.